includes/html/dev-overview-data.inc.php

```diff
@@ -121,11 +121,9 @@
     echo "<div class='row'><div class='col-sm-4'>$uptime_text</div><div class='col-sm-8'>$uptime</div></div>";
 }
 
-if ($device['location_id']) {
+if ($device['location_id'] && $location = Location::find($device['location_id'])) {
```

Comment:
isn't there a reduction possible to:
```
$location = Location::find($device['location_id']);
if ($location) {
```

Reply:
@SourceDoctor That causes an extra SQL query in some cases.

Reply:
ok :-)